AngularLanguageService/AngularLanguageService.csproj

```diff
@@ -65,8 +65,9 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Client" Version="16.10.171" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.VisualStudio.Language.Intellisense" Version="16.10.55" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Client" Version="17.2.2105" PrivateAssets="all" />
```

Comment:
I *think* `PrivateAssets` is only needed for `VS.ExternalApis.WebTools.References` (used for the `Reference` items above).